Formula/ace.rb

```diff
@@ -11,6 +14,9 @@ class Ace < Formula
     sha256 "72f21357797c350fe547a7978f402e564cf328c3662a0e321d1ec293f4899139" => :mavericks
   end
 
+  # this patch backports new files for macOS Sierra and possibility to compile with Xcode 8 on El Capitan
+  patch :DATA
```

Comment:
The problem is that ACE repository is not compilable because of their build system. We can only compile the release they do.
This unfortunately remove the possibility to add `HEAD` to the ACE formula.
This patch has already been merged into the master of ACE, but not in any released version.